app/lib/calculation_output.rb

```diff
@@ -0,0 +1,7 @@
+class CalculationOutput
+  def initialize(capital_collation:)
+    @capital_collation = capital_collation
```

Comment:
Is there a better name for this? I'm not sure that I know what a capital_collation is? Is it not just vehicles and partner vehicles?

Reply:
Good point. I wanted a term to describe the collection of capital subtotals (of which vehicles is the only one populated for now, but I want the name to signpost that this is where additional subtotals should go). `CapitalSubtotals`? `CapitalCalculationOutput`?

Reply:
CapitalSubtotals works for me

Reply:
Or maybe CapitalAssessorOutput, to be clearer about where it's from

Reply:
I guess though it's from both the CapitalCollator and CapitalAssessor, so maybe `CapitalCollatorAndAssessorOutput` even though it's a mouthful.

I'm not keen on christening these things `CapitalCalculation` because we've already got a bunch of Calculator classes which are small bits of the overall calculation. And CapitalSubtotal introduces a new concept, which might confuse

Reply:
I'm a little reticent to tie the name of the object to the name of the class that produced it, only because various things that neither know nor care about CapitalCollator/CapitalAssessor/CapitalCollatorAndAssessor have to interact with these objects. A low-level decorator entity that wants to render a vehicle value subtotal into JSON probably wants to refer to an object that contains that subtotal in terms of what that object is, not where it came from. So I lean towards CapitalSubtotals as it's the most universally relevant, descriptive name. (I think CapitalSummary would possibly be the best name, but for now that's taken).